src/js/views/globals/app-nav/app-nav_views.js

```diff
@@ -95,30 +111,23 @@ const AdminToolsDroplist = Droplist.extend({
   },
 });
 
-const AppNavView = View.extend({
-  className: 'app-nav',
+const BottomNavView = View.extend({
+  className: 'app-nav__bottom',
   regions: {
-    navMain: {
-      el: '[data-nav-main-region]',
-      replaceElement: true,
-    },
-    navContent: '[data-nav-content-region]',
     adminTools: {
       el: '[data-nav-admin-tools-region]',
       replaceElement: true,
     },
   },
-  triggers: {
-    'click .js-add-patient': 'click:addPatient',
-  },
   template: hbs`
-    <div data-nav-main-region></div>
-    <div class="overflow-y" data-nav-content-region></div>
-    <div class="app-nav__bottom">
-      {{#if canPatientCreate}}
-        <div class="flex flex-align-center app-nav__bottom-button js-add-patient">{{fas "circle-plus"}}{{ @intl.globals.appNav.appNavViews.appNavView.addPatient }}</div>
-      {{/if}}
-      <div data-nav-admin-tools-region></div>
+    {{#if canPatientCreate}}
+      <div class="flex flex-align-center app-nav__bottom-button js-add-patient">
+        {{fas "circle-plus"}}{{#unless isMinimized}}<span class="u-text--overflow">{{ @intl.globals.appNav.appNavViews.appNavView.addPatient }}</span>{{/unless}}
+      </div>
+    {{/if}}
+    <div data-nav-admin-tools-region></div>
+    <div class="flex flex-align-center app-nav__bottom-button js-minimize-menu">
+      {{fas "square-caret-left"}}{{#unless isMinimized}}<span class="u-text--overflow">{{ @intl.globals.appNav.appNavViews.appNavView.minimizeMenu }}</span>{{/unless}}
```

Comment:
the icon should switch when minimized

Reply:
Good call, missed that. Updated in recent push.